packages/nextjs/contracts/Constants.ts

```diff
@@ -0,0 +1,218 @@
+const universalEthAddress =
+  "0x49D36570D4E46F48E99674BD3FCC84644DDD6B96F7C741B1562B82F9E004DC7" as const;
+
+const universalStrkAddress =
+  "0x04718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d" as const;
+
+const universalClassHash = '0x046ded64ae2dead6448e247234bab192a9c483644395b66f2155f2614e5804b0' as const;
+
+const universalAbi = [
```

Comment:
> currently ABI in devnet doesnt match ABI on sepolia, due to a previous mismatch between devnet and public networks. But now this should be solved. use ONE single ABI for all tokens.

sir, we should have a unified ABI here, but the ABI configuration in devnet is different from that in Sepolia and other networks. Is the code here using Sepolia's ABI or devnet's ABI?